extensions/extended-blocks/core-embed/core-embed.php

```diff
@@ -0,0 +1,4 @@
+<?php
+
+// Add Loom.com as an oEmbed provider.
+wp_oembed_add_provider( '#https?://(www\.)?loom\.com/share/.*#i', 'https://www.loom.com/v1/oembed', true );
```

Comment:
Do you think you could add a simple test for this, kind of like what we have here?
https://github.com/Automattic/jetpack/blob/master/tests/php/modules/shortcodes/test-class.others.php